tera-templates/rustdoc/page.html

```diff
@@ -0,0 +1,56 @@
+{%- import "macros.html" as macros -%}
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
```

Comment:
Why does this use a new base instead of reusing `base.html`?

Reply:
`base.html` injects a lot of extra style stuff for rustdoc pages, this was the simplest solution